db-engine/src/main/java/com/splicemachine/db/iapi/sql/conn/Authorizer.java

```diff
@@ -94,6 +94,9 @@
 	int CREATE_ROLE_PRIV = 19;
 	int DROP_ROLE_PRIV = 20;
 
+	/* ACCESS schema privilege for multi-tenancy support */
+	int ACCESS_PRIV = 21;
```

Comment:
Why not to name the constant by its purpose, e.g. `MULTI_TENANT_SCHEMA_ACCESS_PRIV`?

Reply:
Though this is mainly implemented for the support of multi-tenancy, it can simply be used to restrict one user's visibility from another user's schema. I'll change the comment instead of the name.